cmake/helpers.cmake

```diff
@@ -36,3 +36,23 @@ function(add_cppformat name)
 
     add_dependencies(cppformat cppformat-${name})
 endfunction()
+
+function(check_cppformat name)
```

Comment:
I don't think this is necessary, we can verify if the code is formatted correctly by just formatting it in CI and looking if that caused any changes. So just like we do it for code generation. See [this line](https://github.com/oneapi-src/unified-runtime/blob/main/CMakeLists.txt#L173) (the docker comment is no longer relevant since we found the pip clang-format package).